content/en/releases/checklist-for-victims-of-a-data-breach.md

```diff
@@ -0,0 +1,187 @@
+
+### Request a prohibition of direct marketing from Traficom and Data & Marketing Association of Finland (ASML)
+
+*   A consumer can limit phone marketing by calling Data & Marketing Association of Finland (ASML) service number 0600 13404 from Monday to Friday between 8 and 18. The service costs 0,39€ / minute + operator service charges. Queuing is free-of-charge. (Note. You can't call from abroad.)
+*   ASML does not charge any other fees from the caller. More information available from [ASML website](https://www.asml.fi/kieltopalvelut/). (currently only in Finnish)
+*   Prohibition of data disclosure from Finnish Transport and Communications Agency Traficom registries can be made from [Traficom website](https://www.traficom.fi/en/services/how-prohibit-disclosure-your-data).
+
+### Your Right to be Forgotten from search results
+
+*   The GDPR grants you the right to request removal of certain personal information from Internet search engines.
+*   [Google Personal Information Removal Request Form](https://www.google.com/webmasters/tools/legal-removal-request?complaint_type=rtbf&hl=en&rd=1)
+*   [Bing Request to Block Bing Search Results In Europe](https://www.bing.com/webmaster/tools/eu-privacy-request)
+
+
+### Take care of yourself
+
+*   Don't panic. It's possible that only some of your data has been leaked.
+*   See below under "_Where to get help under crisis?_" for a list of organizations you can talk with about your issues. You are not alone.
+
+### Other tips
+
+*   Was it your work email instead of your personal email? Contact your employer. For example, talk with your human resources manager. You may get support.
+*   Consider special [non-disclosure for personal safety](https://dvv.fi/en/non-disclosure-for-personal-safety). This is a special protection for cases with a serious threat of physical violence. Talk with the police.
+*   Monitor your digital footprint actively in the future. You can use [Have I been pwned?](https://haveibeenpwned.com/)  -service to notify you if your information has been leaked publicly. The service is maintained by the security expert Troy Hunt. Alternatively, you can use the [Identity Theft Checker from Finnish F-Secure](https://www.f-secure.com/fi/home/free-tools/identity-theft-checker) or the Finnish [Badrap.io](https://badrap.io/) service.
+*   You can use [Google Alerts](https://www.google.fi/alerts) for your own name. The service sends an email if the keyword you provide it appears in Google Search results. Keywords to be used can be e.g. your own phone number, name and email address.
+*   If you receive a registered letter, authenticate it's sender before you sign it as received. The sender might be a criminal who wants to verify your address.
+
+## Where to get help in a crisis?
+
+### Vastaamo crisis phone
+
+*   Contact number: [044 4141 997](tel:+358444141997), open on Monday to Friday 21, Saturday 10 to 14, 17 to 21 and Sunday 10 to 21
```

Comment:
Languages availble?   Finnish, Swedish, English presumably?

Reply:
Presumably. We don't actually know for certain.